modules/distribution/src/main/assembly/bin.xml

```diff
@@ -167,5 +167,12 @@
         <include>*</include>
       </includes>
     </fileSet>
+    <fileSet>
+      <directory>${project.basedir}/../../contrib/grafana</directory>
+      <outputDirectory>conf/grafana</outputDirectory>
```

Comment:
In 3af5d62113339, Dashboard config now gets placed in `contrib/grafana` in the tarball.